common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -369,6 +372,17 @@ open class MusicService : MediaBrowserServiceCompat() {
                 }
             }
         }
+        override fun onPlayerError(error: ExoPlaybackException) {
+            when (error.type) {
+                ExoPlaybackException.TYPE_SOURCE -> {
+                    Toast.makeText(applicationContext,
+                        R.string.error_media_not_found,
+                        Toast.LENGTH_LONG).show()
+                    Log.e(TAG, "TYPE_SOURCE: " + error.sourceException.message)}
+                ExoPlaybackException.TYPE_RENDERER -> Log.e(TAG, "TYPE_RENDERER: " + error.rendererException.message)
+                ExoPlaybackException.TYPE_UNEXPECTED -> Log.e(TAG, "TYPE_UNEXPECTED: " + error.unexpectedException.message)
+            }
```

Comment:
Please add handling for additional exception types: 

```
ExoPlaybackException.TYPE_OUT_OF_MEMORY -> {
                    TODO()
                }
                ExoPlaybackException.TYPE_REMOTE -> {
                    TODO()
                }
```

Reply:
got! I've pushed the changes ;) Many thanks again !!!